nautobot/docs/apps/migrating-jobs-from-nautobot-v1.md

```diff
@@ -0,0 +1,157 @@
+# Migrating Jobs from Nautobot v1.X to Nautobot v2.0
+
+## Quick Summary of Job Class Changes
+
+[`self.run(self, data, commit)` must be changed to include all Job variables](#job-run-signature)
+
+[`self.test_*` and `self.post_run()` methods were removed](#test_-and-post_run-methods)
+
+[`read_only` no longer changes the behavior of Nautobot core](#read-only-meta-attribute)
+
+[`self.job_result` should no longer be modified or saved from within a Job](#job_result)
+
+[Jobs must be registered in the celery task registry](#job-registration)
+
+[`self.failed` removed](#tracking-job-state)
+
+[The job logging methods have been renamed and their signature changed](#job-logging)
+
+[The `request` property has been changed to a celery request instead of a Django request](#request-property)
+
+## Overview
+
+Some fundamental changes were made to Jobs in Nautobot v2.0. This document outlines the changes that were made and how to migrate your existing Jobs to work in Nautobot v2.0. For more information about the changes made to the Job class and Job model in Nautobot v2.0 see the [Upgrading From Nautobot v1](../installation/upgrading-from-nautobot-v1.md#job-changes) documentation.
+
+### Job Package Names
+
+All Jobs are now imported as normal Python packages instead of virtually imported which means that Job file code can be shared with other Jobs or Python modules.
```

Comment:
```suggestion
All Jobs are now imported as normal Python packages, instead of virtually imported, which means that Job file code can be shared with other Jobs or Python modules.
```